graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/MoreSearchAdapterES7.java

```diff
@@ -135,7 +136,14 @@ public MoreSearch.Result eventSearch(String queryString, TimeRange timerange, Se
     }
 
     @Override
-    public void scrollEvents(String queryString, TimeRange timeRange, Set<String> affectedIndices, Set<String> streams, int batchSize, ScrollEventsCallback resultCallback) throws EventProcessorException {
+    public void scrollEvents(String queryString, TimeRange timeRange, Set<String> affectedIndices, Set<String> streams,
+                             int batchSize, ScrollEventsCallback resultCallback) throws EventProcessorException {
+        this.scrollEvents(queryString, timeRange, affectedIndices, streams, Collections.emptyList(), batchSize, resultCallback);
+    }
+
+    @Override
+    public void scrollEvents(String queryString, TimeRange timeRange, Set<String> affectedIndices, Set<String> streams,
+                             List<UsedSearchFilter> filters, int batchSize, ScrollEventsCallback resultCallback) throws EventProcessorException {
```

Comment:
You have added `filters` parameter to this method... but it is never used in the method body...

Reply:
Thanks for noticing this. I forgot the Elasticsearch implementation. I have added it, but I will still need to work on testing this in the morning.

Reply:
The Elasticsearch implementation tests successfully for me.